### PR TITLE
[TW-680] v10 screenshot changes

### PR DESCRIPTION
**Pull request by petewitcher**

### CODE REVIEW

File: src/pages/docs/designing-and-developing-your-api/versioning-an-api/api-versions.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ To publish an API version:
 1. To make this version available on the Private API Network, select **Add to Private API Network**. You can select a folder to place the API version in, or create a new folder.
```
Comment (jkonrath-postman) [inline src/pages/docs/designing-and-developing-your-api/versioning-an-api/api-versions.md:52]: Does this change based on role? It says "Request to add..." in the screenshot.

File: src/pages/docs/getting-started/introduction.md [hunk 2]
```diff
@@ -49,7 +49,7 @@ contextual_links:
 
 Welcome to the Postman docs! This is the place to find official information on how to use Postman in your API projects.
 
-<img alt="Postman full view" src="https://assets.postman.com/postman-docs/v10/intro-postman-request-selected-v10.jpg" width="1100px"/>
+<img alt="Postman full view" src="https://assets.postman.com/postman-docs/v10/intro-postman-request-selected-1-v10.jpg" width="1100px"/>
```
Comment (jkonrath-postman) [inline src/pages/docs/getting-started/introduction.md:52]: Super nitpicky problem here, but these are of the web app, and they were originally the desktop app, so the bottom bar is wrong. (No search/replace, no capture requests.)

Also super minor, but drag up the bottom part with the response so you kill the white space below the parameters and expand the view of the response a little.

File: src/pages/docs/getting-started/navigating-postman.md [hunk 3]
```diff
@@ -53,7 +53,7 @@ Postman has a variety of tools, views, and controls to help you manage your API
     * [Environment selector and environment quick look](#environment-selector-and-environment-quick-look)
 * [Footer](#footer)
 
-<img alt="Postman app" src="https://assets.postman.com/postman-docs/v10/navigating-postman-app-overview-v10.jpg" width="1100px"/>
+<img alt="Postman app" src="https://assets.postman.com/postman-docs/v10/navigating-postman-app-overview-2-v10.jpg" width="1100px"/>
```
Comment (jkonrath-postman) [inline src/pages/docs/getting-started/navigating-postman.md:56]: Same thing on the desktop vs web app version; we should be using desktop here.

File: src/pages/docs/running-collections/working-with-data-files.md [hunk 4]
```diff
@@ -41,11 +41,11 @@ You can select a data file to use in a [collection run](/docs/running-collection
 1. Select your collection and drag it into the **Run Order** work area.
 1. Select your data file with the __Select File__ button.
 
-    <img alt="Data File Select" src="https://assets.postman.com/postman-docs/select-data-file-v8.jpg" height="350px"/>
+    <img alt="Data File Select" src="https://assets.postman.com/postman-docs/v10/select-data-file-v10.jpg" height="350px"/>
```
Comment (jkonrath-postman) [inline src/pages/docs/running-collections/working-with-data-files.md:44]: Fine here but make a note that this will probably changed when SCR ships.

**Review by jkonrath-postman — APPROVED** [review]
A couple of little corrections, but otherwise looks good!

File: src/pages/docs/designing-and-developing-your-api/versioning-an-api/api-versions.md (hunk 1 above)
Comment (petewitcher) [reply src/pages/docs/designing-and-developing-your-api/versioning-an-api/api-versions.md:52]: I'll get access to Basic, Professional, and Enterprise plans to test this and find out.

File: src/pages/docs/getting-started/introduction.md (hunk 2 above)
Comment (petewitcher) [reply src/pages/docs/getting-started/introduction.md:52]: Fixed. I'll watch for the search and replace field to distinguish between the two going forward.

File: src/pages/docs/getting-started/navigating-postman.md (hunk 3 above)
Comment (petewitcher) [reply src/pages/docs/getting-started/navigating-postman.md:56]: Fixed

File: src/pages/docs/running-collections/working-with-data-files.md (hunk 4 above)
Comment (petewitcher) [reply src/pages/docs/running-collections/working-with-data-files.md:44]: Added comment

File: src/pages/docs/designing-and-developing-your-api/versioning-an-api/api-versions.md (hunk 1 above)
Comment (petewitcher) [reply src/pages/docs/designing-and-developing-your-api/versioning-an-api/api-versions.md]: Added `> If your team has enabled the [approval process](/docs/collaborating-in-postman/adding-private-network/#turn-on-the-approval-process) for the Private API network, you can request to add the published API to the Private API Network. Your teams API Network Manager will approve the request.`

Comment (petewitcher) [reply src/pages/docs/designing-and-developing-your-api/versioning-an-api/api-versions.md]: Confirmed, only the API Network Manager has `Add to Private Network`, all other roles have `Request to Add...`

**Pull request merged**